src/main/java/appeng/services/VersionChecker.java

```diff
@@ -76,16 +76,24 @@ public void run()
 	{
 		Thread.yield();
 
-		final String rawLastCheck = this.config.lastCheck();
-		final long lastCheck = Long.parseLong( rawLastCheck );
-		final Date now = new Date();
-		final long nowInMs = now.getTime();
-		final long intervalInMs = this.config.interval() * SEC_TO_HOUR * MS_TO_SEC;
-		final long lastAfterInterval = lastCheck + intervalInMs;
+		try
+		{
+			final String rawLastCheck = this.config.lastCheck();
+			final long lastCheck = Long.parseLong( rawLastCheck );
+			final Date now = new Date();
+			final long nowInMs = now.getTime();
+			final long intervalInMs = this.config.interval() * SEC_TO_HOUR * MS_TO_SEC;
+			final long lastAfterInterval = lastCheck + intervalInMs;
+
+			this.processInterval( nowInMs, lastAfterInterval );
 
-		this.processInterval( nowInMs, lastAfterInterval );
+			AELog.info( "Stopping AE2 VersionChecker" );
+		}
+		catch( Exception e )
```

Comment:
main thread as in log "Exception in thread ?" as well as log the exception to `stdout`.
This is just to prevent the jvm from reporting any unhandled exception, but still log the exception in the right context (and not simply `stdout`)